scripts/populate_sample_contributor_data_test.py

```diff
@@ -221,11 +221,15 @@ def _assert_topics_in_classroom(self, classroom_name: str) -> None:
         classroom_dict = self.get_json(
             '%s/%s' % (feconf.CLASSROOM_DATA_HANDLER, classroom_name))
         topic_summaries = topic_fetchers.get_all_topic_summaries()
-        for topic_summary in topic_summaries:
+        topic_summary_dicts_from_classroom = classroom_dict[
+            'topic_summary_dicts']
+        for index, topic_summary in enumerate(topic_summaries):
             topic_summary_dict = dict(topic_summary.to_dict())
-            topic_summary_dict['is_published'] = False
-            self.assertIn(
-                topic_summary_dict, classroom_dict['topic_summary_dicts'])
+            topic_summary_dict_from_classroom = (
+                topic_summary_dicts_from_classroom[index])
+            topic_summary_dict['is_published'] = (
+                topic_summary_dict_from_classroom['is_published'])
+            assert topic_summary_dict, topic_summary_dict_from_classroom
```

Comment:
Is this the right way to assert equality? I think it will never fail, right?) In general, use methods from `unittest` module like assertEqual, assertIn etc.

Reply:
@DubeySandeep Done.